data/json/items/tools.json

```diff
@@ -2940,38 +2940,218 @@
     "use_action": "MOLOTOV_LIT"
   },
   {
-    "id": "acidbomb",
-    "type": "TOOL",
+    "id": "acidbomb_large",
+    "type": "AMMO",
+    "category": "weapons",
+    "ammo_type" : "thrown",
     "symbol": "*",
-    "color": "yellow",
-    "name": "acid bomb",
-    "description": "This is a fragile container filled with acid.  Throw it to spill out a pool of potent acid.",
-    "price": 5000,
-    "material": "glass",
-    "flags": "ACT_ON_RANGED_HIT",
-    "weight": 650,
+    "color": "i_green",
+    "name": "large acidbomb",
+    "description": "There is enough frothing fluid inside this acidbomb to coat a relatively large area with fields of acid.",
+    "price": 0,
+    "material": ["glass"],
+    "weight": 1640,
     "volume": 2,
-    "bashing": 4,
+    "bashing": 8,
     "to_hit": -1,
+    "flags": ["ACT_ON_RANGED_HIT", "CUSTOM_EXPLOSION"],
+    "use_action": {
+        "type": "transform",
+        "target": "acidbomb_large_act",
+        "target_charges": 0,
+        "active": true
+    }
+  },
+  {
+    "id": "acidbomb_large_act",
+    "type": "TOOL",
+    "symbol": "*",
+    "color": "i_green",
+    "name": "large acidbomb",
+    "description": "A thrown acidbomb, in the process of loosing its liquid.",
+    "price": 0,
+    "material": ["glass"],
+    "weight": 1640,
+    "volume": 1,
+    "bashing": 8,
+    "to_hit": -1,
+    "max_charges": 1,
+    "initial_charges": 1,
+    "turns_per_charge": 1,
+    "flags": ["BOMB"],
     "category": "weapons",
-    "use_action": "ACIDBOMB_ACT"
+    "use_action": {
+        "type": "explosion",
+        "power": 1,
```

Comment:
Why is it exploding? No need for an explosion when you just want to spread acid.